src/algorithms/algorithms.hpp

```diff
@@ -8,9 +8,11 @@
 #include <cmath>
 #include <functional>
 
+#include "help_functions.hpp"
```

Comment:
```suggestion
#include "common.hpp"
```
мб так лучше? у нас все общее на работе выносят в пакет/файл common